src/Compilers/Core/Portable/Serialization/SimpleRecordingObjectBinder.cs

```diff
@@ -10,20 +10,20 @@ namespace Roslyn.Utilities
     /// <summary>
     /// A binder that gathers type/reader mappings during object writing
     /// </summary>
-    internal sealed class SimpleRecordingObjectBinder : RecordingObjectBinder
+    internal sealed class SimpleRecordingObjectBinder : ObjectBinder
```

Comment:
This isn't new.  It was already part of the code base.  The concurrent binder is used for trees, since it is shared for all tree serialization.

---
In reply to: [84797760](https://github.com/dotnet/roslyn/pull/14671#discussion_r84797760) [](ancestors = 84797760)